### PR TITLE
FRO 181 targets updates

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Add filtering and sorting to targets

### CODE REVIEW

File: src/utils/sortFilterTargets.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  /* 
+    Net zero targets should be first, 
+    then GHG targets, 
+    then legislative (legal) targets,
+    then targets in the economy-wide sector
+  */
+  t = t.sort((a, b) => {
+    if (a[netZeroKey] === "TRUE" && b[netZeroKey] === "FALSE") {
+      return -1;
```
Comment (diversemix) [inline src/utils/sortFilterTargets.ts:31]: Again just style we could ease reading with:
```
const a_after_b = 1;
const a_before_b = -1;
```

**Review by diversemix — APPROVED** [review]
lgtm

File: src/utils/sortFilterTargets.ts (hunk 1 above)
Comment (PatFawbertMills) [reply src/utils/sortFilterTargets.ts:31]: It is fairly standard sort syntax but I'm happy to make the change

Comment (diversemix) [reply src/utils/sortFilterTargets.ts:31]: Its not a request to make a change mearly a suggestion

**Pull request merged**